.github/workflows/release-github.yml

```diff
@@ -0,0 +1,56 @@
+# Copyright The Ratify Authors.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# workflow created based on verbiage from https://github.com/notaryproject/notation/blob/v1.2.0/.github/workflows/release-github.yml
+name: release-github
+
+on:
+  push:
+    tags:
+      - v*
+
+permissions:
+  contents: read
+
+jobs:
+  build:
+    permissions:
+      contents: write  # for goreleaser/goreleaser-action to create a GitHub release
+    name: Release Ratify CLI Binaries
+    runs-on: ubuntu-20.04
```

Comment:
Is Ubuntu 20.04 too old?

Reply:
Also it is reaching the end of LTS

Reply:
We may also need to update for notation

Reply:
Updated.

Reply:
https://github.com/notaryproject/notation/issues/1139

Added an issue for notation